app/controllers/propertyrentals/income/IncomeFromPropertyController.scala

```diff
@@ -26,7 +26,7 @@ import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import repositories.SessionRepository
 import service.SessionService
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
-import views.html.propertyrentals.income.IncomeFromPropertyView
```

Comment:
I had renamed from **_IncomeFromPropertyRentalsView_** to **_IncomeFromPropertyView_** because this view no longer is being used for the rentals journey but also for the combined journey and that's why I removed the Rentals part in the name of the view

Reply:
I see. Makes sense @cmantzoukashmrc